Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Corefinement/intersection_callbacks.h

```diff
@@ -78,6 +79,7 @@ class Collect_face_bbox_per_edge_bbox_with_coplanar_handling {
   const VertexPointMapE& vpmap_tme;
   EdgeToFaces& edge_to_faces;
   CoplanarFaceSet& coplanar_faces;
+  const Visitor& visitor;
```

Comment:
Don't we store a copy?

Reply:
the callback use the one passed by the coref code. No need to copy it.